### ISSUE TITLE
`org-bullets`, `org-bullets-mode` may be a compatible inclusion

### ISSUE THREAD

**Issue by jasonhemann** [amno1/org-view-mode#14]
I find the `org-bullets` package and org-bullets mode a nice way to do a slight bit of some of this cleanup. Worth considering as an option/inclusion/compatibility mode?

**Comment by amno1** [comment]
Doesn't it just replace stars in headings for utf-8 characters? We are hiding stars in view-mode; bullets are not visible in view-mode anyway.

I have it installed, and I see no difference when I turn on org-view-mode. Am I missing something?

**Comment by jasonhemann** [comment]
Replaces multiple starts with a single UTF-8 heading. I like to see *some*, but not all of the formatting. If that's not compatible with org-view-mode's goals, that's a fine answer.

If you want to see it turn on, you have to also `(require 'org-bullets-mode)` within a standard org-mode configuration.

**Comment by amno1** [comment]
I concluded that color coding for headings is enough to understand the hierarchy. Each heading level is a different color. 

However, there is an option to customize org-view-hide-stars. Set it to nil, and if you normally have org-bullets-mode on, it will show you org-bullets and the rest will be hidden. I just tested that it works, see [screenshot](https://ibb.co/FnDZWmZ). 

If you want org-bullet-mode only in combination with org-view-mode, then perhaps add hook to org-view-mode in  your init file:

`(add-hook 'org-view-mode-hook (lambda () (org-bullets-mode +1)))`

Actually something like this is better:

```
(add-hook 'org-view-mode-hook
            (lambda ()
              (if org-view-mode
                  (org-bullets-mode +1)
                (org-bullets-mode -1))) nil t)
```

You will of course have to (require 'org-bullets) somewhere in your init file; either in you use-package config for org-view-mode, or in with-eval-after-load somewhere in your init file.

I don't have any special goals for this; just to declutter as much as possible, but it's Emacs; there are like zillion packages and options. Much of 3rd party options have alternatives, and everything is so subjective. Some things are better left to users :-).

I don't use org-bullets anymore, I found they slow my org-mode a bit, so I opt out, but I have an old computer.

I hope that helps. Best regards and thanks for the input.
